### PR TITLE
(DOCS-6594) Fleet Automation

### PR DESCRIPTION
**Pull request by hestonhoffman**
### What does this PR do? What is the motivation?
New page for Fleet Automation

### Merge instructions
Targeting Friday for merge

### CODE REVIEW

File: content/en/agent/fleet_automation/_index.md [hunk 1]
```diff
@@ -0,0 +1,87 @@
...
+Fleet Automation is available to all users in a Datadog organization. You can control access to specific functionality:
+
+| Permission | Description |
+|--------------|---------------|
+| `API keys read`| Restricts which users can view and search agents by API key. |
+| `Agent flare collection` | Restricts which users can remotely send flares. |
+
+For information on setting up roles and permissions, see [Access Control][5].
+
+## Further reading
```
Comment (hestonhoffman) [inline content/en/agent/fleet_automation/_index.md:79]:
```suggestion
## Further Reading
```

File: content/en/agent/fleet_automation/_index.md [hunk 2]
```diff
@@ -0,0 +1,87 @@
...
+further_reading:
+- link: "/agent/remote_config"
+  tag: "Documentation"
+  text: "Find out more about Remote Configuration"
+- link: "/infrastructure/list/#agent-configuration"
+  tag: "Documentation"
+  text: "Learn about the Agent configuration view"
+---
+
+{{< callout btn_hidden="true">}}Fleet Automation is in private beta.{{< /callout >}}
```
Comment (urseberry) [inline content/en/agent/fleet_automation/_index.md:14]: maybe use the shortcode option to change the title? "Join the Beta!" looks funny when there's no way to actually do so on the page.

File: content/en/agent/fleet_automation/_index.md [hunk 3]
```diff
@@ -0,0 +1,87 @@
...
+## Overview
+
+Datadog Fleet Automation allows you to gain visibility into your fleet of Agents to ensure they are set up correctly and optimized to meet your evolving observability needs.
+
+{{< img src="agent/fleet_automation/fleet_automation1.png" alt="The fleet automation page." style="width:100%;" >}}
+
+With Fleet Automation, you can:
+- Reduce the time it takes to debug an issue by sending a remote flare straight from the [Fleet Automation][1] page.
+- Ensure you're running up-to-date Agents with the latest security fixes and feature enhancements by identifying outdated Agent versions.
+- Identify which Agents are using a particular API Key so you can safely rotate keys and disable old keys without impact.
```
Comment (urseberry) [inline content/en/agent/fleet_automation/_index.md:29]:
```suggestion
- Identify which Agents are using a particular API key so you can safely rotate keys and disable old keys without impact.
```

File: content/en/agent/fleet_automation/_index.md [hunk 4]
```diff
@@ -0,0 +1,87 @@
...
+- View Agent and Agent integration configurations to help confirm deployment changes and troubleshoot any configuration related issues.
+
+To access Fleet Automation, click **Integrations** > [**Fleet Automation**][1].
+
+From the Fleet Automation page, you can gain insight into unmonitored Agents, Agents that need to be updated, or Agents that have integration issues. For each Agent, you can see:
```
Comment (urseberry) [inline content/en/agent/fleet_automation/_index.md:34]:
```suggestion
Use the Fleet Automation page to gain insight into unmonitored Agents, Agents that need to be updated, or Agents that have integration issues. For each Agent, you can see:
```

File: content/en/agent/fleet_automation/_index.md [hunk 5]
```diff
@@ -0,0 +1,87 @@
...
+- **Remote Configuration**: For information on supported Agent versions and configuration steps, see [Enabling Remote Configuration][3].
+- **Agent configuration**: Agent version 7.39/6.39 or later is required to enable the Agent configuration tab. It is enabled by default in Agent versions 7.47.0/6.47.0 or later. To enable Agent configuration manually, set `inventories_configuration_enabled` in your [Agent configuration file][2] to `true`. Alternatively, use the `DD_INVENTORIES_CONFIGURATION_ENABLED` environment variable.
+- **Agent integration configuration**: Agent integration configuration is enabled by default on Agent versions 7.49/6.49 or later. To enable Agent integration configuration manually, set `inventories_checks_configuration_enabled` in your [Agent configuration file][2] to `true`. Alternatively, use the environment variable `DD_INVENTORIES_CHECKS_CONFIGURATION_ENABLED`.
+
+Datadog recommends upgrading your Agents regularly to make sure you have access to the latest features and security updates.
+
+## Send a remote flare
+
+Before you send a flare, make sure that [Remote Configuration is enabled on the selected Agent](#configuring-fleet-automation).
```
Comment (urseberry) [inline content/en/agent/fleet_automation/_index.md:58]:
```suggestion
Before you send a flare, make sure that Remote Configuration is [enabled](#configuring-fleet-automation) on the selected Agent.
```

Comment (urseberry) [reply content/en/agent/fleet_automation/_index.md:58]: The length of the hyperlinked phrase is definitely a matter of personal preference. My preference is for shorter links.

File: content/en/agent/fleet_automation/_index.md [hunk 6]
```diff
@@ -0,0 +1,87 @@
...
+
+To send a remote flare:
+1. From the [Fleet Automation][1] page, select an Agent that requires support.
+1. Click **Support**.
+1. Click **Send Support Ticket**.
+1. Provide an existing Zendesk support ticket number. If you don't provide a ticket number, one is created on your behalf.
+1. Enable **Debug mode** to allow Datadog support staff to troubleshoot your issue faster. The log level is reset to its previous configuration after the flare is sent.
```
Comment (urseberry) [inline content/en/agent/fleet_automation/_index.md:65]:
```suggestion
1. Enable **Debug mode** to allow Datadog support staff to troubleshoot your issue faster. The log level is reset to its previous configuration after you send the flare.
```

File: content/en/agent/fleet_automation/_index.md [hunk 7]
```diff
@@ -0,0 +1,87 @@
...
+1. Click **Send Ticket**.
+
+## Control access to Fleet Automation
+
+Fleet Automation is available to all users in a Datadog organization. You can control access to specific functionality:
+
+| Permission | Description |
```
Comment (urseberry) [inline content/en/agent/fleet_automation/_index.md:72]: A+ use of a small table to present the information clearly.

File: content/en/agent/fleet_automation/_index.md (hunk 2 above)
Comment (hestonhoffman) [reply content/en/agent/fleet_automation/_index.md:14]: Yeah, I agree. I'd turned the title off previously, but it looked weird with so little text in the box. I'll think about this some more

**Review by urseberry — APPROVED** [review]

File: content/en/agent/fleet_automation/_index.md [hunk 8]
```diff
@@ -0,0 +1,89 @@
...
+<div class="alert alert-warning">Fleet Automation is not supported for your selected <a href="/getting_started/site">Datadog site</a> ({{< region-param key="dd_site_name" >}}).</div>
+{{< /site-region >}}
+
+## Overview
+
+Datadog Fleet Automation allows you to centrally govern and remotely manage Datadog Agents at scale to support your evolving observability needs.
+
+{{< img src="agent/fleet_automation/fleet-automation-view.png" alt="The fleet automation page" style="width:100%;" >}}
+
+With Fleet Automation platform, you can:
```
Comment (urseberry) [inline content/en/agent/fleet_automation/_index.md:26]:
```suggestion
With the Fleet Automation platform, you can:
```

File: content/en/agent/fleet_automation/_index.md [hunk 9]
```diff
@@ -0,0 +1,89 @@
...
+- View Agent and Agent integration configurations to help confirm deployment changes and ensure configuration consistency.
+- Easily send a flare from within your organization, reducing the time it takes to debug issues on an Agent.
```
Comment (urseberry) [inline content/en/agent/fleet_automation/_index.md:28]:
```suggestion
- Send a flare from within your organization, reducing the time it takes to debug issues on an Agent.
```

File: content/en/agent/fleet_automation/_index.md [hunk 10]
```diff
@@ -0,0 +1,89 @@
...
+- The Agent's Remote Configuration status
+- The products that are enabled on the Agent
+
+Selecting an Agent gives you more information about it, including its configuration, connected integrations, and a support tab that you can use to send a remote flare.
+
+{{< img src="agent/fleet_automation/fleet-automation-yaml.png" alt="An Agent's integration information" style="width:100%;" >}}
+
+## Configuring Fleet Automation
+
+Fleet Automation incorporates several Datadog features, which are all enabled automatically in Agents version 7.49/6.49 or later. To ensure you have access to all of the features, upgrade your Agents to version 7.49/6.49 or later.
```
Comment (urseberry) [inline content/en/agent/fleet_automation/_index.md:47]:
```suggestion
Fleet Automation incorporates several Datadog features, which are all enabled automatically in Agent version 7.49/6.49 or later. To ensure you have access to all of the features, upgrade your Agents to version 7.49/6.49 or later.
```

File: content/en/agent/fleet_automation/_index.md [hunk 11]
```diff
@@ -0,0 +1,89 @@
...
+
+{{< img src="agent/fleet_automation/fleet-automation-flare.png" alt="The Send Ticket button launches a form to send a flare for an existing or new support ticket" style="width:100%;" >}}
+
+## Control access to Fleet Automation
+
+Fleet Automation is available to all users in a Datadog organization. You can control access to specific functionality:
+
+| Permission | Description |
+|--------------|---------------|
+| `API keys read`| Restricts which users can view and search agents by API key. |
```
Comment (urseberry) [inline content/en/agent/fleet_automation/_index.md:76]:
```suggestion
| `API keys read`| Restricts which users can view and search Agents by API key. |
```

**Review by urseberry — APPROVED** [review]
Just a few nits.

**Pull request merged**